### DISCUSSION TITLE
Do you support AWS GLUE schema registry?

### DISCUSSION THREAD

**Discussion by alberttwong** [MaterializeInc/datagen#120] [General]
Do you support AWS GLUE schema registry?

**Comment by alberttwong** [comment]
It looks like no. https://repost.aws/questions/QUkzoERVKBRSiOfwUMz6plZA/endpoint-to-access-aws-glue-schema-registry

**Comment by bobbyiliev** [comment]
Hey!

As far as I know, you cannot use AWS Glue Schema Registry as a drop-in replacement for Confluent Schema Registry. Currently, the datagen CLI only supports Confluent Schema Registry.

**Reply by morsapaes** [reply]
To add to Bobby's answer: only schema registries that are API-compatible with the Confluent Schema Registry are supported as is (like, for example, the Redpanda Schema Registry). Unfortunately, Glue Schema Registry does not offer this level of interoperability. We're open to contributions, though, if you're interested in adding support for it, @alberttwong!
